source/develop/release-management/features/storage/qcow2v3.html.md

```diff
@@ -0,0 +1,73 @@
+---
+title: qcow2v3 image format
+category: feature
+authors: mlipchuk
+feature_name: qcow2v3 image format
+feature_modules: engine,vdsm
+feature_status: Implementation
+---
+
+# QCOW2V3 Image Format
+
+
+## Overview
+
+The QCOW image format includes a version number to allow introducing new features that change the image format in an incompatible way.
+Newer QEMU versions support a new format of QCow2 version 3, which is not backwards compatible and introduce
```

Comment:
Be consistent: it's QCOW, in all caps, not QCow.
Also, it's worth mentioning what "newer" QEMU versions are (1.7 and above)